packages/hoprd/src/commands/utils/cliOptions.ts

```diff
@@ -1,7 +1,6 @@
 // allowed CLI options
 // @ts-ignore
 export const cli_options: string[][] = [
-  ['-b', '--bootstrapNode', undefined, 'starts HOPR as a bootstrap node'],
```

Comment:
There should be a flag that tells `hoprd` to register its own (hopefully public) ipv4 address

Reply:
`--announce` does this